python/doc/user_manual/optimization.rst

```diff
@@ -41,13 +41,3 @@ Optimization solvers
     Pagmo
     SQP
     TNC
-
-Nearest point assessment
-========================
-
-.. autosummary::
-    :toctree: _generated/
-    :template: class.rst_t
-
-    NearestPointChecker
-    NearestPointCheckerResult
```

Comment:
maybe NearestPointChecker can be removed completely from the python api (else its docstring will be reported as missing)

Reply:
Yes: I have already done it! In a previous commit (dc75618) which title is:
"Removed the visibility of section "Nearest point assessment" in the user manual](https://github.com/openturns/openturns/pull/2405/commits/dc756188f72a63d5e053fa5ce7bb2e63d80b2c9c)

Reply:
I meant removing it completely from swig

Reply:
ok... I do not know what to do more....

Reply:
ok, it can be done later

Reply:
I finally removed them completely...